### PR TITLE
Delete suppressions against resolved issues

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/tests/issues.targets [hunk 1]
```diff
@@ -1184,14 +1184,8 @@
         <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/StaticVirtualMethods/InterfaceVariance/**">
             <Issue>https://github.com/dotnet/runtime/issues/88689</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Regressions/coreclr/GitHub_85240/test85240/**">
-            <Issue>https://github.com/dotnet/runtime/issues/71095</Issue>
```
Comment (ivanpovazan) [inline src/tests/issues.targets:1188]: The runtime test `test85240` is still failing: https://dev.azure.com/dnceng-public/public/_build/results?buildId=568219&view=logs&j=28035387-23c9-50dd-d519-66c274947a2c&t=ea523488-181f-5d56-a79f-8dbe85d021ad&l=34
I think the reason for this is that Mono does not handle fptr type equality and compatibility.
I suggest we leave the test disabled, but update the reason to: https://github.com/dotnet/runtime/issues/90308

File: src/tests/issues.targets [hunk 2]
```diff
@@ -1184,14 +1184,8 @@
...
-        </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/opt/Remainder/Regressions/Regression1/**">
-            <Issue>https://github.com/dotnet/runtime/issues/79022</Issue>
-        </ExcludeList>
```
Comment (ivanpovazan) [inline src/tests/issues.targets:1192]: This is correct. `JIT/opt/Remainder/Regressions/Regression1` is now passing with Mono.
Thank you for cleaning this up.

File: src/tests/issues.targets (hunk 1 above)
Comment (jkotas) [reply src/tests/issues.targets:1188]: Fixed. Thank you!

**Review by ivanpovazan — APPROVED** [review]
LGTM, thank you.

**Pull request merged**